tests/drivers/flash_patch/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8.2)
```

Comment:
I think a newer version is used in upstream zephyr, you could check that.

Reply:
You're right 3.13.1. Seems NCS samples are still on 3.8.x though. I'll change.